### PR TITLE
Add more Galaxy xml tags

### PR DESCRIPTION
**Pull request by khillion**
Hi Eric,

I have added few new classes of tags that were not implemented in the library: 
* `<options>` with `<filter>` and `<column>`
* `<container>` for `<requirements>`
* Started to add `<tests>` section
* `<section>` in `<inputs>`
* `<collection>` with `<discover_datasets>` in `<outputs>`

I did not work on the link between parameters and command yet, but I need these classes to improve import from an existing XML to try to cover all possibilities. Thanks!

### CODE REVIEW

File: examples/tool.xml [hunk 1]
```diff
@@ -11,20 +15,19 @@
...
-    <param argument="-float" help="Float help" label="Float label" name="float" type="float" value="0"/>
+    <section name="float_section" title="Float section">
+      <param argument="-float" help="Float help" label="Float label" name="float" type="float" value="0"/>
+    </section>
     <conditional argument="cond" label="Conditional" name="cond">
-      <param argument="Select" label="Author did not provide help for this parameter... " name="Select" type="select">
-        <option value="bye">2</option>
-        <option value="hi">1</option>
-      </param>
+      <param argument="Select" label="Author did not provide help for this parameter... " name="Select" type="select"/>
```
Comment (hexylena) [inline examples/tool.xml]: Looks like something went wrong with the select since it isn't outputting the options any more

**Review by hexylena — COMMENTED** [review]
Awesome progress! This is super exciting

File: examples/tool.xml (hunk 1 above)
Comment (khillion) [reply examples/tool.xml]: Thanks, I did not notice. It should be fixed now, I let you check and merge the pull request if everything is fine :).

**Comment by hexylena** [comment]
Nice! :D

**Pull request merged**